t/055-subreq-vars.t

```diff
@@ -65,8 +65,8 @@ M(http-subrequest-start) {
 --- request
 GET /lua
 --- response_body_like: 500 Internal Server Error
---- error_log chop
-variable "dog" cannot be assigned a value (maybe you forgot to define it first?)
+--- error_log eval
+qr/variable "(dog|cat)" cannot be assigned a value \(maybe you forgot to define it first\?\)/
```

Comment:
@maage Oh yeah, I was wrong. The variable sets happen when the subrequest is created instead of first being referenced in the subrequest running phases. I forgot the implementation details. Sorry. You're right.